initialize_internal_tide.jl

```diff
@@ -17,26 +17,29 @@ function initialize_internal_tide(
     Nx,
     Ny,
     Nz;
+    Δtᵒ,
+    tᶠ,
+    θ,
+    U₀,
+    N,
     closure = SmagorinskyLilly(),
-    θ = 3.6e-3,
-    U₀ = 0.025,
+    output_mode,
```

Comment:
These should have default values, e.g. `output_mode = "test"` and `output_writer = true`. Actually, everything after the `;` in the argument list should have a default value.

If for some reason you don't want to set a default value, then those arguments should appear before the `;`. But I think most of these should have a default value that is the value used in the reference simulation.